src/Bundle/test/config/packages/test/winzou_state_machine.yaml

```diff
@@ -32,3 +32,18 @@ winzou_state_machine:
             reopen:
                 from: [ closed ]
                 to: review
+    subscription:
+        class: App\Subscription\Entity\Subscription
+        property_path: state
```

Comment:
Minor thing,  but we are mixing `state` and `currentState` variables. Property is called state, but getter/setter are for `currentState`. TBH, I'm surprised this configuration even work, I would expect exception here, as there is no `setState` method on Subscription

Reply:
Hum indeed, I think this works cause my property is public. I'm gonna try to change that.